src/Stratis.Bitcoin.Features.MemoryPool/MempoolSignaled.cs

```diff
@@ -128,8 +128,8 @@ public void Start()
                     await behaviour.SendTrickleAsync().ConfigureAwait(false);
             },
             this.nodeLifetime.ApplicationStopping,
-            repeatEvery: TimeSpans.TenSeconds,
-            startAfter: TimeSpans.TenSeconds);
+            repeatEvery: TimeSpans.FiveSeconds,
+            startAfter: TimeSpans.FiveSeconds);
```

Comment:
I think the start after should still be 10 seconds (even more) so we give the node enough time to start